liquidator/test/Liquidator.js

```diff
@@ -168,8 +168,115 @@ contract("Liquidator.js", function(accounts) {
     // Sponsor2 should have zero collateral left in their position from the liquidation.
     assert.equal((await emp.getCollateral(sponsor2)).rawValue, 0);
 
-    //Sponsor3 should have all their collateral left and no liquidations.
+    // Sponsor3 should have all their collateral left and no liquidations.
```

Comment:
This should not have gotten through the linter. I added this to our config and the PR is [here](https://github.com/UMAprotocol/protocol/pull/1081).